src/utilities/globals/index.ts

```diff
@@ -3,10 +3,8 @@ import { invariant, InvariantError } from "ts-invariant";
 // Just in case the graphql package switches from process.env.NODE_ENV to
 // __DEV__, make sure __DEV__ is polyfilled before importing graphql.
 import DEV from "./DEV";
-export { DEV }
-export function checkDEV() {
-  invariant("boolean" === typeof DEV, DEV);
-}
+export { DEV };
```

Comment:
What do you think about removing the above `import` and renaming in the `export`?
```suggestion
export  { default as __DEV__} from "./DEV";
```

Reply:
Got rid of the `getDEV` function: https://github.com/apollographql/apollo-client/pull/10521/commits/28ba4e7aa44caa346e71647f361673e5a9b7ec35